Makefile

```diff
@@ -129,17 +133,8 @@ main-test-status:
 DOCS_DIR = docs
 DOCS_BUILD_DIR = docs/_build
 
-# Common ERA5 data for nb-tests and doctests
+# Cache common ERA5 data for nb-tests and doctests in DiskCacheStore(cache_dir=".doc-test-cache")
 ensure-era5-cached:
-	python -c 'from pycontrails.datalib.ecmwf import ERA5; \
```

Comment:
Consolidate `cache-era5-gcp` and `ensure-era5-cached` into one call with a common DiskCacheStore